wcfapp/WcfServiceWebApp/Web.config

```diff
@@ -0,0 +1,50 @@
+﻿<?xml version="1.0"?>
+<configuration>
+
+  <appSettings>
+    <add key="aspnet:UseTaskFriendlySynchronizationContext" value="true" />
+  </appSettings>
+  <system.web>
+    <compilation debug="true" targetFramework="4.6.2" />
+    <httpRuntime targetFramework="4.6.2"/>
+  </system.web>
+  <system.serviceModel>
+    <services>
+      <service name="WcfServiceWebApp.Service1">
+        <endpoint binding="basicHttpBinding" contract="WcfServiceWebApp.IService1" />
+        <endpoint binding="netTcpBinding" contract="WcfServiceWebApp.IService1" bindingConfiguration="noSecurityBind"/>
+        <endpoint address="mex" binding="mexTcpBinding" contract="IMetadataExchange" />
+      </service>
+    </services>
+    <bindings>
+      <netTcpBinding>
+        <binding name="noSecurityBind" portSharingEnabled="false">
+          <security mode="None" />
+        </binding>
+      </netTcpBinding>
+    </bindings>
+    <behaviors>
+      <serviceBehaviors>
+        <behavior>
+          <!-- To avoid disclosing metadata information, set the values below to false before deployment -->
+          <serviceMetadata httpGetEnabled="true" httpsGetEnabled="true"/>
+          <!-- To receive exception details in faults for debugging purposes, set the value below to true.  Set to false before deployment to avoid disclosing exception information -->
+          <serviceDebug includeExceptionDetailInFaults="false"/>
+        </behavior>
+      </serviceBehaviors>
+    </behaviors>
+    <protocolMapping>
+        <add binding="basicHttpsBinding" scheme="https" />
+    </protocolMapping>    
+    <serviceHostingEnvironment aspNetCompatibilityEnabled="true" multipleSiteBindingsEnabled="true" />
```

Comment:
>aspNetCompatibilityEnabled="true" [](start = 31, length = 33)

Did you test this with net.tcp? The documentation for aspNetCompatibilityEnabled says this:
> When this attribute is set to true, requests to Windows Communication Foundation (WCF) services flow through the ASP.NET HTTP pipeline, and communication over non-HTTP protocols is prohibited.

This along with the portSharingEnabled=false suggests you didn't test net.tcp with this.

Reply:
I would not put up anything here without testing!

Everything worked for me. This config file is generated from VS template. I try to keep the changes minimum so it's clearer to users what changes are necessary.

It's interesting to see the discrepancy of the doc and testing here. Something we should understand better later.